CMakeLists.txt

```diff
@@ -56,15 +56,10 @@ STRING(REGEX REPLACE "\-.*$" "" VERSION_NUM "${PROJECT_VERSION_FULL}")
 # PROJECT_VERSION_PATCH, libopenshot-audio_VERSION_PATCH
 PROJECT(libopenshot-audio LANGUAGES C CXX VERSION ${VERSION_NUM})
 
-# JuceHeader.h needs a hexadecimal version number for the project
-if(CMAKE_VERSION VERSION_LESS 3.13)
-	set(PROJECT_VERSION_HEX ${HEX_VERSION_OVERRIDE}) # Ugly hardcoding
-else()
-	math(EXPR PROJECT_VERSION_HEX 
-		"(${PROJECT_VERSION_MAJOR} << 16) + \
```

Comment:
You deleted these protections for CMake < 3.13, that's why the builds failed.

Reply:
Yup, whoops... mistake.